agent/balrogagent/log.py

```diff
@@ -20,28 +20,43 @@ class JsonLogFormatter(logging.Formatter):
     Adapted from:
     https://github.com/mozilla-services/mozservices/blob/master/mozsvc/util.py#L106
     """
+
     LOGGING_FORMAT_VERSION = "2.0"
 
     # Map from Python logging to Syslog severity levels
-    SYSLOG_LEVEL_MAP = {
-        50: 2,  # CRITICAL
-        40: 3,  # ERROR
-        30: 4,  # WARNING
-        20: 6,  # INFO
-        10: 7,  # DEBUG
-    }
+    SYSLOG_LEVEL_MAP = {50: 2, 40: 3, 30: 4, 20: 6, 10: 7}  # CRITICAL  # ERROR  # WARNING  # INFO  # DEBUG
```

Comment:
This is an unfortunate transformation. I wonder if using the # fmt: off pragma here would be appropriate? Or introduce constants for CRITICAL, ERROR, etc.? Or just leave it alone? :)

Reply:
Yeah, this is awful. It looks like Black doesn't handle this correctly even with a shorter line length. I filed https://github.com/ambv/black/issues/800 for it. We need to do something in the meantime though - this is horribly unreadable with this change.

Reply:
Or use the constants from the logging module?